homeassistant/components/device_tracker/__init__.py

```diff
@@ -41,14 +41,16 @@
     SOURCE_TYPE_BLUETOOTH,
     SOURCE_TYPE_GPS,
     SOURCE_TYPE_ROUTER,
+    SOURCE_TYPE_ZIGBEE,
```

Comment:
Does this have any special implications? Eg in the person component, will we handle zigbee in some special way? Should we instead make a general device source type, or should each new device that doesn't fit into bluetooth, gps or router add a new device type?

Reply:
I just followed the pattern that was emerging. I don’t have strong feelings either way.

Reply:
Hard to say, Im torn as well for UniFi, should it really be Router in my case. What does it really do? Is it cosmetic? Then Zigbee could be nice, but should I then do WiFi and Wired for source types instead?

Reply:
I think we should only use source type to allow builtin logic, eg in person component, to take different actions based on source type. It shouldn't be a cosmetic. If we don't need to differentiate zigbee in builtin logic, we don't need a special type for this device, I'm reasoning. But we might need a type for other devices than the other three types, in the future?

Up until now we have used the router type for all scanner based device trackers.

Reply:
so what do you want me to do?

Reply:
I'd play conservative and use `SOURCE_TYPE_ROUTER` for now. We can change it later when we know better.